### ISSUE TITLE
Re-share file to a conversation

### ISSUE THREAD

**Issue by mario** [nextcloud/spreed#1183]
Use case: 

- I attach/share an image to a conversation
- Months later, I want to share the same image so it appears as a new message but I can't do that

**Comment by nickvergessen** [comment]
Yeah, sharing to the same target twice is not allowed, but we could indeed just trigger the message again without adding a second share.

**Comment by mario** [comment]
This would be good, yes :)
